.github/codecov.yml

```diff
@@ -1,2 +1,3 @@
 ignore:
   - "**/*_bindings.cpp"  # ignore pybind11 bindings
+  - "pyvrp/cli.py"  # tested in other ways than unit tests
```

Comment:
I think this is OK, since we're going to be using the CLI ourselves a lot anyway, and it's not really a functional, standalone component.